### PR TITLE
HTML-16 - evenNumbers

### PR DESCRIPTION
**Pull request by zeljkosoko**

### CODE REVIEW

File: evenNumbers.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+    for (let i = 1; i < 100; i++) {
+        if (i % 2 == 0) {
+            nums.push(i);
+        }
+    }
+}
+evenNumbers();
+
+console.log(nums);
```
Comment (hanii1993) [inline evenNumbers.js:13]: Add new line

**Review by hanii1993 — CHANGES_REQUESTED** [review]

File: evenNumbers.js (hunk 1 above)
Comment (zeljkosoko) [reply evenNumbers.js:13]: New empty line at the end?

Comment (hanii1993) [reply evenNumbers.js:13]: yes

Comment (zeljkosoko) [reply evenNumbers.js:13]: empty lines added.

**Review by hanii1993 — APPROVED** [review]

**Review by ban17 — APPROVED** [review]

**Pull request merged**